src/GNDStk/Tree/src/find.hpp

```diff
@@ -0,0 +1,97 @@
+
+// -----------------------------------------------------------------------------
+// Functions for FINDING THINGS in trees
+// meta
+// child
+// operator()
+// -----------------------------------------------------------------------------
+
+/*
+Summary:
+
+Tree::meta()
+   decltype(auto) meta  ( const string          &str ) const;
+   decltype(auto) meta  ( const meta_t <RESULT> &kwd ) const;
+
+Tree::child()
+   decltype(auto) child ( const string          &str ) const;
+   decltype(auto) child ( const child_t<RESULT> &kwd ) const;
+
+Tree::operator()()
+   decltype(auto) operator() ( const meta_t <RESULT> &kwd             ) const;
+   decltype(auto) operator() ( const child_t<RESULT> &kwd             ) const;
+   decltype(auto) operator() ( const child_t<RESULT> &kwd, Ts &&...ts ) const;
+*/
+
+// fixme Probably have that bool &found stuff in these as well...
+
+
+
+// -----------------------------------------------------------------------------
+// meta
+// -----------------------------------------------------------------------------
+
+decltype(auto) meta(const std::string &str) const
```

Comment:
OK. I thought it would make sense to have meta(), child(), and operator()() in the same file, insofar as they're all intended for finding stuff in the tree, but I'll go ahead and split them out.